src/animation/AnimationMixer.js

```diff
@@ -83,8 +83,10 @@ Object.assign( THREE.AnimationMixer.prototype, THREE.EventDispatcher.prototype,
 
 		var root = optionalRoot || this._root,
 			rootUuid = root.uuid,
-			clipName = ( typeof clip === 'string' ) ? clip : clip.name,
-			actionsForClip = this._actionsByClip[ clipName ];
+			clipObject = typeof clip === 'string' ? this.getClipByName( clip ) : clip,
```

Comment:
The way Mixer was intended to be used is to have one per character.  Thus you can tell a character to walk and it would run the appropriate clip for that character as it would be preloaded into its Mixer under that name.

We have sort of diverged from that intended design to a global Mixer, which really wasn't intended.  I allowed for a Mixer to have multiple roots so you could animate something in a characters hand -- it would be a separate root.  Not really so that you could have a centralized Mixer that controls multiple characters.

Unity and UE4 have one mixer per character.  And they even have more complex structures built on the per-character mixers to make more complex things.  I am little concerned that if we continue down this global mixer path we'll continue to run into design issues like this one as this isn't how things are usually done.